nesta/production/orms/meetup_es_config.json

```diff
@@ -0,0 +1,132 @@
+{
+    "mappings": {
+        "_doc": {
+            "dynamic": "strict",
+            "properties": {
+                "coordinate_of_group": {
+                    "type": "geo_point"
+                },
+                "count_member_group": {
+                    "type": "integer"
+                },
+                "datetime_birth_group": {
+                    "format": "yyyy-MM-dd",
+                    "type": "date"
+                },
+                "id_continent_group": {
+                    "type": "keyword"
+                },
+                "id_country_group": {
+                    "type": "keyword"
+                },
+                "id_iso2_group": {
+                    "type": "keyword"
+                },
+                "id_iso3_group": {
+                    "type": "keyword"
+                },
+                "id_isoNumeric_group": {
+                    "type": "integer"
+                },
+                "id_of_group": {
+                    "type": "keyword"
+                },
+                "id_state_group": {
+                    "type": "keyword"
+                },
+                "name_of_category": {
+                    "type": "keyword"
+                },
+                "name_of_group": {
+                    "fields": {
+                        "keyword": {
+                            "type": "keyword"
+                        }
+                    },
+                    "type": "text"
+                },
+                "placeName_city_group": {
+                    "fields": {
+                        "keyword": {
+                            "type": "keyword"
+                        }
+                    },
+                    "type": "text"
+                },
+                "placeName_continent_group": {
+                    "fields": {
+                        "keyword": {
+                            "type": "keyword"
+                        }
+                    },
+                    "type": "text"
+                },
+                "placeName_country_group": {
+                    "fields": {
+                        "keyword": {
+                            "type": "keyword"
+                        }
+                    },
+                    "type": "text"
+                },
+                "placeName_state_group": {
+                    "fields": {
+                        "keyword": {
+                            "type": "keyword"
+                        }
+                    },
+                    "type": "text"
+                },
+                "terms_memberOrigin_group": {
```

Comment:
it's a field that doesn't exist yet. I'm going to add a field describing the country origins of the members, above a threshold. For example, if 90% of the members are from France, 8% are from Germany and 2% from elsewhere, a threshold of 5% would mean that `terms_memberOrigin_group` would be equal to `['FR', 'DE']` which would allow you to map the 'internationalness' of the group. This is briefly indicated in the task list of issue #96 

I'm going to prefix the field name with `_terms` to clarify that this is in development.

Reply:
wondering if these can be [ES objects](https://www.elastic.co/guide/en/elasticsearch/reference/6.6/object.html), like `{FR: 0.9, DE: 0.08}`

Reply:
certainly could be - perhaps both fields would be useful from a performance perspective

Reply:
like adding a field `object_memberOrigin_group` (object) on top of `terms_memberOrigin_group` (array) ?

Reply:
yep - I've just added these to issue #96. Good to sign off?